### DISCUSSION TITLE
Using Custom Operator Approach

### DISCUSSION THREAD

**Discussion by singhavishek** [netket/netket#1274] [Q&A]
I want to define a custom operator for following,
<psi | H | psi> =  < psi | sum_<i,j> sigma_i . Sigma_j | psi>

I looked into the custom operator approach defined in [https://netket.readthedocs.io/en/latest/advanced/custom_operators.html](url).

Can some one refer me to a code or reference where a custom operator like above is defined

**Comment by PhilipVinc** [comment]
Hi @singhavishek , sorry for the delay I was on holiday.

The operator you write there can be written by hand by summing some standard LocalOperators.
Still, if you want to write it yourself, I suggest you download that web page (https://netket.readthedocs.io/en/latest/advanced/custom_operators.html) as it's actually a notebook and try to run it yourself.

You should be able to tweak the leanoperator which implements X  

To download the notebook just click on the download icon on the top navber.
